src/builders/grover_algorithm_builder.py

```diff
@@ -1,37 +1,23 @@
+import os
 from qiskit import QuantumCircuit
+from qiskit.circuit.library import PhaseOracle, GroverOperator
 
 
 class GroverAlgorithmBuilder:
     def __init__(self):
-        self.oracle_quantum_circuit = None
-        self.grover_quantum_circuit = None
-        self.reflection_quantum_circuit = None
+        self.number_of_qubits = None
 
-    def two_qubits_oracle_builder(self):
-        number_of_qubits = 2
-        quantum_circuit = QuantumCircuit(number_of_qubits, name='oracle')
-        quantum_circuit.cz(0, 1)
-        quantum_circuit.to_gate()
-        self.oracle_quantum_circuit = quantum_circuit
+        self.oracle = None
```

Comment:
The name `oracle` is quite vague and must be more accurate. With `oracle_quantum_circuit` we have a better clue the variable contains a quantum circuit.

```suggestion
        self.oracle_quantum_circuit = None
```